src/input/tablet.c

```diff
@@ -224,15 +263,15 @@ handle_axis(struct wl_listener *listener, void *data)
 	struct drawing_tablet *tablet = ev->tablet->data;
 	struct drawing_tablet_tool *tool = ev->tool->data;
 
-	if (!tool_supports_absolute_motion(ev->tool)) {
-		return;
-	}
-
+	tablet->dx = 0;
```

Comment:
Note to self: no need to remember dx, dy since they are relative. May be the same applies for tilt?